main.go

```diff
@@ -53,7 +117,8 @@ func helpFunc(commands map[string]cli.CommandFactory) cli.HelpFunc {
 	// aren't shown in any help output. We use this for prerelease functionality
 	// or advanced features.
 	hidden := map[string]struct{}{
-		"exec": {},
+		"exec":   {},
+		"server": {},
```

Comment:
This is the old entrypoint for the k8s controller.

Since we're actually releasing the CLI at some point, we want to hide this, as no one should be leveraging it. I'm _almost_ tempted to make a breaking change and rename the entrypoint to something slightly more awkward/verbose like `run-kubernetes-controller` and `exec` to `run-kubernetes-deployment` since only the helm chart and internal deployment components should ever invoke these. If that happens though, we'll need to document that the helm chart would only support specific versions of api-gateway going forward, so I figured we probably just want to leave the names as is for now.

Reply:
I'd prefer leaving the names as-is for now with everything else we have in flux; however, I've been eyeballing at least renaming the `server` and `exec` packages in a similar way to what you mentioned.